zerver/lib/slack_data_to_zulip_data.py

```diff
@@ -521,23 +521,47 @@ def convert_slack_workspace_messages(slack_data_dir: str, users: List[ZerverFiel
 
     logging.info('######### IMPORTING MESSAGES STARTED #########\n')
 
-    zerver_message, zerver_usermessage, attachment, uploads, \
-        reactions = channel_message_to_zerver_message(realm_id, users, added_users,
-                                                      added_recipient, all_messages,
-                                                      zerver_realmemoji,
-                                                      realm['zerver_subscription'],
-                                                      added_channels,
-                                                      domain_name)
-
-    message_json = dict(
-        zerver_message=zerver_message,
-        zerver_usermessage=zerver_usermessage)
-    message_filename = os.path.join(output_dir, "messages-000001.json")
-    logging.info("Writing Messages to %s\n" % (message_filename,))
-    create_converted_data_files(message_json, output_dir, '/messages-000001.json')
+    total_reactions = []  # type: List[ZerverFieldsT]
+    total_attachments = []  # type: List[ZerverFieldsT]
+    total_uploads = []  # type: List[ZerverFieldsT]
+
+    message_id = usermessage_id = reaction_id = attachment_id = 0
+    id_list = [message_id, usermessage_id, reaction_id, attachment_id]
```

Comment:
This initial value doesn't seem to be used; are you just doing this to set the type?

Reply:
I proposed and we agreed to treat this as opaque data, since it is essentially just state that the function requires storing and re-supplying between calls.